### PR TITLE
Back end event sign in controller update

### PR DESCRIPTION
**Pull request by rhrussell**
Did some testing but make sure you guys test as well just to be safe. There are no offenses with Rubocop and code coverage is still at 100%, but make sure you guys read the code just to make sure I did not miss any edge cases when it comes to checking into an event with the code and the tests.

Testing link: https://bmes-member-back-end-ev-onawxl.herokuapp.com/

### CODE REVIEW

**Review by varvind — COMMENTED** [review]
CSV feature, repeating events, and individual event seem to work fine, still need to test user profile and events late at night

**Review by varvind — COMMENTED** [review]
Details page, event sign in and user profile work well

File: app/admin/events.rb [hunk 1]
```diff
@@ -134,13 +135,13 @@ def create
                                 newevent['endtime(3i)'].to_i, newevent['endtime(4i)'].to_i,
                                 newevent['endtime(5i)'].to_i)
       # gets the weeks parameter from new event
+      newstarttime += 5.hours
```
Comment (varvind) [inline app/admin/events.rb:138]: probably want to add a comment here on why we chose to do this

**Review by varvind — CHANGES_REQUESTED** [review]
Nice, code looks good, just one small thing to add and ill continue to test

File: spec/routes_spec.rb [hunk 2]
```diff
@@ -5,52 +5,47 @@
...
+      #                        starttime: '2025-01-03 00:00:00', endtime: '2025-01-03 00:00:00',
+      #                        eventpass: 'pass3', eventtype: 'General Meeting')
       event1.save
-      event2.save
+      # event2.save
       visit events_path
 
       path1 = "//a[@href='/events/new?event_id=" + event1.id.to_s + "']"
-      path2 = "//a[@href='/events/new?event_id=" + event2.id.to_s + "']"
+      # path2 = "//a[@href='/events/new?event_id=" + event2.id.to_s + "']"
```
Comment (burdick-alex) [inline spec/routes_spec.rb:38]: Be sure to uncomment or delete these comments before finishing and merging

File: spec/routes_spec.rb [hunk 3]
```diff
@@ -5,52 +5,47 @@
...
       find(:xpath, path1).click
       sleep(2)
       click_link('Back to List')
       sleep(2)
-      find(:xpath, path2).click
-      click_link('Back to List')
+      # find(:xpath, path2).click
```
Comment (burdick-alex) [inline spec/routes_spec.rb:43]: Be sure to uncomment or delete these comments before finishing and merging

File: spec/routes_spec.rb [hunk 4]
```diff
@@ -5,52 +5,47 @@
...
+      # click_link('Back to List')
 
       sleep(2)
       event1.destroy
-      event2.destroy
+      # event2.destroy
```
Comment (burdick-alex) [inline spec/routes_spec.rb:48]: Be sure to uncomment or delete these comments before finishing and merging

**Review by burdick-alex — CHANGES_REQUESTED** [review]
Doesn't seem to be working on heroku, but some minor things. Either uncomment these lines or remove them before merging

**Review by varvind — APPROVED** [review]
Nice job!

**Review by burdick-alex — APPROVED** [review]
looks good
